tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js

```diff
@@ -593,7 +588,7 @@ describe('BO - Orders - View and edit order : Check payment Block', async () =>
     it('should add payment when amount is equal to the rest to the new invoice', async function () {
       await testContext.addContextItem(this, 'testIdentifier', 'testAmountSupTotal', baseContext);
 
-      const invoice = `#IN0000${invoiceID >= 10 ? '' : '0'}${invoiceID + 1}`;
+      const invoice = `#IN0000${invoiceID + 1 >= 10 ? '' : '0'}${invoiceID + 1}`;
```

Comment:
```suggestion
      const invoice = `#IN0000${(invoiceID + 1) >= 10 ? '' : '0'}${invoiceID + 1}`;
```

Reply:
Not mandatory, bring no value to add parenthesis :sweat_smile: